app/components/DepositWithdraw/blocktrades/WithdrawModalBlocktrades.jsx

```diff
@@ -231,15 +232,10 @@ class WithdrawModalBlocktrades extends React.Component {
                     real: amount
                 });
                 
-                let userAmount = new Asset({
-                    amount: this.props.balance.get("balance"),
-                    asset_id: this.props.asset.get("id"),
-                    precision: this.props.asset.get("precision")
-                });
-
-                console.log("Sending Amount: " + sendAmount.amount + " Available Amount: " + userAmount.amount);
+                let userBalance = balance.get("balance");
+                let isUserBalanceEnough = sendAmount.amount < userBalance;
```

Comment:
Please the Asset class apis, `getAmount` and `setAmount` instead of accessing the variables directly.